WorkloadManagementSystem/PilotAgent/pilotTools.py

```diff
@@ -1,6 +1,7 @@
 """ A set of common tools to be used in pilot commands
 """
 
+from __future__ import print_function
```

Comment:
@fstagni Is it still required that the pilotTools are running on pre python 2.6? Because there are run with the local python before DIRAC is installed, right? Pre 2.6 this `from __future__ ...` does not work

Reply:
Good catch. Yes, they do, so changes to these files should be done carefully.

And BTW: @petricm every change that is correct and validated here for Pilot2 files, should be ported to Pilot repository too.

Reply:
I my memory we talked that with v7 pilot2 diapers and we have only pilot3?

Reply:
Is there a clever way, like conditional import, that can make print statement compatible for pythons from 2.6 to 3.0 ? Then this can be added to pythonTools. If not, then pilot stuff should be excluded from the python 3.0 compatibility campaign. We can also have 2 versions of pythonTools (legacy and python 3.0 compatible) and use one or another depending on the python version.

Reply:
One could replace the print by logging, https://docs.python.org/2/library/logging.html, exists since python 2.3

Reply:
I have reverted PilotAgent and excluded the directory from the test for now. As for the conditional import it is tricky since the `from __future__ imports` must occur at the beginning of the file.

Reply:
Well Pilot somehow has to be compatible with python 2.6, 2.7 and 3. I don't plan to do it in this repository anyway.

Reply:
if 2.6 is the earliest, then there is no problem, if we were to support something before that (2.5, 2.4?) there would be issues